packages/button-theme/src/Callout.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { applyTheme, StyleConfig } from '@button-inc/component-library/Callout';
+import styles from './styles';
+
+const config: StyleConfig = {
+  defaultProps: {
+    variant: 'primary',
+    size: 'medium',
+  },
+  staticProps: [],
+};
+
+export const BaseCallout = applyTheme(styles, config);
+
+export default function Callout(props: any) {
+  const { content, children, ...rest } = props;
+
+  return <BaseCallout {...rest}>{content ? <p>{content}</p> : children}</BaseCallout>;
```

Comment:
_related issue:_ It's odd that the Storybook control for `content` doesn't work, as this line says that content should take priority. 🤔

Reply:
I think most of this is coming from how the story is set up; I think I can fix at least some of it

Reply:
Yeah, I noticed in Storybook too that some components have _Template_ sub-items in addition to _Default_ and _HTML_, but can't see a clear reason why. Figured it was something with the Storybook setup that I might not see/understand.

Reply:
> Other than the new bug/issue pointed out above, the Button themed component is in line with the BCGov one and reflects the core component. Storybook is rendering just fine too! taco

I updated the bug card with some info on how to possibly fix it in the bcgov theme